aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ReadHandler.java

```diff
@@ -1,10 +1,6 @@
 package software.amazon.ssm.patchbaseline;
 
-import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
-import software.amazon.cloudformation.proxy.Logger;
-import software.amazon.cloudformation.proxy.ProgressEvent;
-import software.amazon.cloudformation.proxy.OperationStatus;
-import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.proxy.*;
```

Comment:
i think it's good to import only those being used

Reply:
??

Reply:
As mentioned the i have modified the changes & imported the needed class instead of entire package load.